src/styles/index.css

```diff
@@ -0,0 +1,70 @@
+.stopwatch__circle,
+.stopwatch__second,
+.stopwatch__min,
+.stopwatch,
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@keyframes hand-rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+body {
+  margin: 0;
+  height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  color: #000;
+  border: 1vmin;
+  border-style: dotted;
+  border-radius: 50%;
+}
+
+.stopwatch__min {
+  width: 3vmin;
+  height: 20vmin;
+  background-color: #0700ff;
+  animation: hand-rotate 3600s infinite steps(60);
+  bottom: 50%;
+  transform-origin: bottom;
+  position: absolute;
+}
+
+.stopwatch__second {
+  width: 1.5vmin;
+  height: 38vmin;
+  background-color: #2c8000;
+  animation: hand-rotate 60s infinite linear;
+  bottom: 50%;
+  transform-origin: bottom;
+  position: absolute;
+}
+
+.stopwatch__circle {
+  width: 5vmin;
+  height: 5vmin;
+  background-color: #f6a603;
+  position: absolute;
+  border-radius: 50%;
+}
```

Comment:
The `position: absolute;` property is duplicated once more. Please refer to the previous comments.